.github/workflows/mirror.yml

```diff
@@ -0,0 +1,36 @@
+name: mirror
```

Comment:
Can we rename it please? Something more descriptive. `mirror-selenium-releases.yml`?

Reply:
Done.